st2chatops/files/st2chatops.env

```diff
@@ -0,0 +1,133 @@
+export st2_hostname="${st2_hostname:-localhost}"
+
```

Comment:
I decided not to remove the env file just for the troubleshooting. It will be removed later on